input/data/2012/2012ApJ...754L..10A/tev-000030-lc.ecsv

```diff
@@ -0,0 +1,26 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - {name: e_min, unit: TeV, datatype: float64}
+# - {name: time, unit: MJD, datatype: float64}
+# - {name: flux, unit: ph / (cm2 s), datatype: float64}
+# - {name: flux_err, unit: ph / (cm2 s), datatype: float64}
+# meta: !!omap
+# - data_type: lc
+# - source_id: 30
+# - source_name: HESS J0632+057
+# - paper_id: 2012ApJ...754L..10A
```

Comment:
We've changed from `paper_id` -> `reference_id` everywhere recently, because not all datasets will be from papers. Please also do the change for all the new files you're adding here.